maintainers/working_groups/learning_journey/tutorial-workflow.md

```diff
@@ -0,0 +1,81 @@
+# Tutorial Workflow
+
+This document describes the recommended workflow when creating tutorials for the learning journey tutorial series.
+
+## Writing process
+### Pick a tutorial
+There are [tracking issues][tracking_issues] for tutorials that the working group has decided should exist as part of the tutorial series.
+Pick an issue that covers a topic that you're either knowledgeable about or have a particular interest in, then comment on the issue indicating your interest in working on the tutorial.
+
+The tracking issue will contain status information such as whether PRs have been submitted to create an outline or a draft of the article or whether someone else has already started work on the tutorial.
+Make sure to read the issue fully to ensure that you don't duplicate work that someone else is already doing!
+
+[tracking_issues]: https://github.com/NixOS/nix.dev/issues?q=is%3Aissue+is%3Aopen+label%3Atracking+label%3A%22learning+journey%22
+
+### Submit an outline PR
+Work on the tutorial series will take place in the `learning-journey` branch.
+Submit a PR with an outline of the tutorial using the [tutorial outline template][tutorial_outline_template], making sure to submit the PR against the `learning-journey` branch.
+The outline should be placed in the [tutorial outlines directory][outlines_dir].
+The outline will go through a review process to ensure that the material is satisfactory and is detailed enough that any other contributor could pick it up and work on it.
+In particular, the tutorial outline should:
+
+- Be a tutorial as defined in the [Diataxis][diataxis_tutorial] framework.
+- Clearly state the prerequisite knowledge.
+- Have a well-defined scope and learning objective.
+
+[tutorial_outline_template]: ./tutorial-outlines/template.md
+[outlines_dir]: ./tutorial-outlines/
+[diataxis]: https://diataxis.fr/tutorials/
+
+### Submit a draft PR
+Once the outline PR has been merged, work can begin on a draft of the tutorial.
+Submit a PR with a draft of the tutorial using the [tutorial template][tutorial_template], placing the draft in the `source/tutorials/learning-journey`.
```

Comment:
We already have this here, in much more detail: https://nix.dev/contributing/writing-a-tutorial

Reply:
I'll link to that. The intention for this document was to have all the steps in one place, so even if some of the information in this document duplicates information, at least it's concise and small enough to fit in your head. In this case I'm convinced the "how to write a tutorial" document is valuable enough to link to rather than paraphase and include inline here.